### PR TITLE
[zh] Translate section of `Define a gRPC liveness probe` into Simplified Chinese

### PR DESCRIPTION
**Pull request by cucker0**
* Synchronize the section of `Define a gRPC liveness probe`  to file content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md.
* Translate section of  `Define a gRPC liveness probe`  into Simplified Chinese on file content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md.
* Copy content/en/examples/pods/probe/grpc-liveness.yaml as content/zh/examples/pods/probe/grpc-liveness.yaml.

/assign @chenrui333

### CODE REVIEW

**Comment by tengqm** [comment]
@cucker0 Please sign the CLA in order for the PR to be reviewed.

**Comment by avinashupadhya99** [comment]
/retitle [zh] Translate section of `Define a gRPC liveness probe` into Simplified Chinese

**Review by Arhell — COMMENTED** [review]
/check-cla

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+  Be sure to configure your gRPC endpoint to listen on the Pod's IP address.
+- Built-in probes do not support any authentication parameters (like `-tls`).
+- There are no error codes for built-in probes. All errors are considered as probe failures.
+- If `ExecProbeTimeout` feature gate is set to `false`, grpc-health-probe does **not** respect the `timeoutSeconds` setting (which defaults to 1s),
+  while built-in probe would fail on timeout.
+-->
+在Kubernetes 1.23之前，通常用 [grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/) 来实现gRPC健康探测，
+正如博客所述 [Health checking gRPC servers on Kubernetes](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/)。
+内置gRPC探测器的行为 类似与 通过grpc-health-probe实现的探测器的行为。
+当从grpc-health-probe探测器 迁移到 内置探测器时，请记住以下的区别：
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:416]:
```suggestion
当从 grpc-health-probe 探测器迁移到内置探测器时，请记住以下区别：
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+
+- 内置探测器是相对于pod IP地址运行的，而 grpc-health-probe 探测器通常是相对于 `127.0.0.1` 运行的。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:418]:
```suggestion
- 内置探针是针对 Pod IP 地址运行的，而 grpc-health-probe 探针通常是针对 `127.0.0.1` 运行的。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+  确保将gRPC endpoint配置为侦听Pod的IP地址。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:419]:
```suggestion
  确保将 gRPC 端点配置为在 Pod 的 IP 地址上监听。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 4]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+- 内置探测器不支持任何身份验证参数（如：`-tls`）。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:420]:
```suggestion
- 内置探针不支持任何身份验证参数（如：`-tls`）。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 5]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+- 内置探测器中没有代码错误。所有的错误被视为探测失败。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:421]:
```suggestion
- 内置探针不支持错误代码；所有的错误都将被视为探测失败。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 6]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+- 如果`ExecProbeTimeout`特性门控设置为`false`，grpc-health-probe将**不再**遵循`timeoutSeconds`的设置（默认为1秒），
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:422]:
```suggestion
- 如果 `ExecProbeTimeout` 特性门控设置为 `false`，grpc-health-probe 将**不再**遵循
  `timeoutSeconds`的设置（默认为 1 秒）。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 7]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+  而内置探测器会在超时时失败。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:423]:
```suggestion
  而内置探针会在超时时失败。
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 8]
```diff
@@ -333,6 +333,95 @@ After 15 seconds, view Pod events to verify that liveness probes:
...
+
+<!--
+Here is an example manifest:
+
+{{< codenew file="pods/probe/grpc-liveness.yaml">}}
+
+To use a gRPC probe, `port` must be configured. If the health endpoint is configured
+on a non-default service, you must also specify the `service`.
+-->
+这有个示例如下：
```
[the remark this answers is not in the document]
Comment (cucker0) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:365]: Here is an example manifest 
翻译为 “下面是一个示例清单” ，感觉好生硬。

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 5 above)
Comment (cucker0) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:421]: 这个翻译得比我原来的要准确。
点赞！

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:365]: "这有个实例如下” 不符合中文表达习惯 ...

**Comment by tengqm** [comment]
@cucker0 Any help needed on this?

**Comment by tengqm** [comment]
We haven't heard a thing from the author for over a month.
/close
Please feel free to reopen and resume the work.

**Pull request closed without merging**